vignettes/vaccineff.Rmd

```diff
@@ -22,143 +22,98 @@ library(vaccineff)
-
-- Proportion of the population vaccinated. Data can be administered as individuals with complete vaccination, individuals without complete vaccination, or as the proportion of the population vaccinated.
-
-- Individuals' demographic information (e.g., sex, age, health insurance) and other variables that may be specified depending on the confounding variables to be introduced in the model.
+- Individuals' demographic information (e.g., sex, age group, health status): These can be used as confounding variables to match the population and reduce observational bias.
 
 ## Modeling vaccine effectiveness
 
 ### VE for Cohort design
 
-The current release of the package bases the estimation of $VE$ in the cohort design on the assumption of proportional hazards between vaccinated and unvaccinated populations. The hazard ratio is estimated using the Cox proportional hazards model implemented in the R package `{survival}`.
+The current release of the package bases the estimation of $VE$ in the cohort design on the assumption of proportional hazards between vaccinated and unvaccinated populations. The $HR$ is estimated using the Cox proportional hazards model implemented in the R package `{survival}`.
 
 The integrated dataset `cohortdata` serves as a minimal example of the package's input. The data is accessed using `data("cohortdata")`.
 
-The following example estimates $VE$ by:
+`vaccineff` has three main functions:
+
 
-1. Defining the immunization information using the function `make_immunization()`. This method returns the cohort data with the immunization date based on the characteristic response time to vaccination (`immunization_delay`) and the specified limits (`outcome_date_col, censoring_date_col, end_cohort`).
+1. `make_vaccineff_data`: This function returns an S3 object of the class `vaccineff_data` with the relevant information for the study. This function also allows to create a matched cohort to control for confounding variables by setting `match = TRUE` and passing the corresponding `exact` and `nearest` arguments.
+`make_vaccineff_data`  supports the method `summary()` to check the characteristics of the cohort, the matching balance and the sizes of matched, excluded, and removed populations.
 
-2. Matching the cohort by exact and nearest confounders using the function `match_cohort()`. This function supports methods like `summary()` to check matching balance and the sizes of matched, excluded, and removed populations, and `get_dataset()` to extract the matched cohort.
+2.  `plot_coverage`: This function returns a plot of the vaccine coverage or the cumulative coverage. If the population is matched, the plot also includes the resulting count of doses after matching. 
```

Comment:
Santi, I think it would be better if the coverage function were an external function, separate from the plot function, in terms of responsibility separation

Reply:
The coverage functions is indeed a separated function. The name is `coh_coverage()`. It is an internal function, as the package is not intended to provide this information, but only to visualize it.